cfht2caom2/instruments.py

```diff
@@ -880,10 +887,35 @@ def _semi_deep_copy_plane(
                     cc.copy_chunk(chunk)
                 )
 
+    def make_axes_consistent(self):
+        pass
```

Comment:
I'm just wondering if these methods shouldn't throw a not-implemented exception to force subclasses to consider them. It's a design pattern which name escapes me right now...

Reply:
I'll updated the code where it didn't break unit tests to do this, and will do this going forward.